test/regress/regress0/strings/Makefile.am

```diff
@@ -100,6 +99,7 @@ EXTRA_DIST +=
 
 #norn-dis-0707-3.smt2
 #norn-ab.smt2
+#type002.smt2
```

Comment:
nit: If we are not going to use it, maybe delete it?

Reply:
At some point, I'll make another pass to either re-enable or delete some of the strings regressions (there a few listed there that are disabled)